packages/cozy-doctypes/src/banking/BankingReconciliator.spec.js

```diff
@@ -20,6 +20,15 @@ describe('banking reconciliator', () => {
     reconciliator = new BankingReconciliator({ BankAccount, BankTransaction })
   })
 
+  afterEach(() => {
+    jest.restoreAllMocks()
+  })
+
+  const fmtCreateOrUpdateCall = call => {
+    const doc = call[0]
+    return doc.label
+  }
```

Comment:
I find it much easier to read diffs from snapshots when the snapshots are formatted correctly in a readable manner.